src/teamwork.sh

```diff
@@ -32,21 +32,57 @@ teamwork::add_comment() {
   log::message "$response"
 }
 
+teamwork::add_tag() {
+  local -r tag_name=$1
+
+  if [ "$ENV" == "test" ]; then
+    log::message "Test - Simulate request. Task ID: $TEAMWORK_TASK_ID - Tag: ${tag_name//\"/}"
+    return
+  fi
+
+  response=$(curl -X "PUT" "$TEAMWORK_URI/projects/api/v1/tasks/$TEAMWORK_TASK_ID/tags.json" \
+       -u "$TEAMWORK_API_TOKEN"':' \
+       -H 'Content-Type: application/json; charset=utf-8' \
+       -d "{ \"tags\": { \"content\": \"${tag_name//\"/}\" } }" )
+
+  log::message "$response"
+}
+
+teamwork::remove_tag() {
+  local -r tag_name=$1
+
+  if [ "$ENV" == "test" ]; then
+    log::message "Test - Simulate request. Task ID: $TEAMWORK_TASK_ID - Tag: ${tag_name//\"/}"
+    return
+  fi
+
+  response=$(curl -X "PUT" "$TEAMWORK_URI/projects/api/v1/tasks/$TEAMWORK_TASK_ID/tags.json" \
+       -u "$TEAMWORK_API_TOKEN"':' \
+       -H 'Content-Type: application/json; charset=utf-8' \
+       -d "{ \"tags\": { \"content\": \"${tag_name//\"/}\" },\"removeProvidedTags\":\"true\" }" )
+
+  log::message "$response"
+}
+
 teamwork::pull_request_opened() {
   local -r pr_url=$(github::get_pr_url)
   local -r pr_title=$(github::get_pr_title)
   local -r user=$(github::get_sender_user)
   local -r pr_stats=$(github::get_pr_patch_stats)
+  local -r pr_body=$(github::get_pr_body)
   IFS=" " read -r -a pr_stats_array <<< "$pr_stats"
 
   teamwork::add_comment "
 **$user** opened a PR: **$pr_title**
 [$pr_url]($pr_url)
-
+---
+  ${pr_body//###/####} 
```

Comment:
So replace all hading level 3 for level 4 headings isn't it?
Why? Just curious

Reply:
So, it's getting anything from 3, 4, 5 to 4,5..

I hope we are not using 2... And just because lvl 3 looks good in github, but way too big in Projects 😊